app/Console/Kernel.php

```diff
@@ -34,6 +35,6 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        //
+        $schedule->command(ForceDeleteTestRecords::class, ['--force'])->hourly();
```

Comment:
do you know if this is going to work on heroku? I know we have to update a cron configuration on the server but I haven't really looked into it.

Reply:
I _think_ we need to add this addon https://devcenter.heroku.com/articles/scheduler

And just doing a quick search, I found some other people who have blogged about the set up:

http://chandrewz.github.io/blog/laravel-and-scheduled-jobs-on-heroku
https://www.neontsunami.com/posts/laravel-scheduler-on-heroku

Happy to look into this with you once it is on thor!

Reply:
these are super helpful - thank you @sbsmith86 !